frontend_tests/node_tests/compose.js

```diff
@@ -109,7 +108,16 @@ function initialize_handlers({override}) {
     compose.initialize();
 }
 
+function mock_banners() {
+    // zjquery doesn't support `remove`, which is used when clearing the compose box.
+    // TODO: improve how we test this so that we don't have to mock things like this.
+    for (const classname of Object.values(compose_banner.CLASSNAMES)) {
+        $(`#compose_banners .${classname}`).remove = noop;
```

Comment:
There are a lot of places where I do something like `#compose_banners .${classname}` and I'm not a huge fan of that `.` being outside of the classname, but I ended up doing this because I wanted to pass `compose_banner.CLASSNAMES.____` directly as the classname to the template, where it shouldn't have the dot. I'm curious if others have thoughts on ways to do this more nicely.